cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -259,3 +261,48 @@ def test_reset_report_menus(appliance, get_custom_report, group, report_menus):
     assert not appliance.collections.reports.instantiate(
         type=folder, subtype=subfolder, menu_name=get_custom_report.menu_name
     ).exists
+
+
+@pytest.mark.parametrize("group", GROUPS)
+@pytest.mark.meta(coverage=[1762363])
+def test_custom_reports_menu_crd(
```

Comment:
Please use `test_custom_reports_menu_crud`

Reply:
I wasn't updating anything here, which is why I went with `crd`.